### PR TITLE
chores(ci): use the ci to prettify modified files

### PR DESCRIPTION
**Pull request by ncomont**
**What is the problem this PR is trying to solve?**

The CI doesn't prettify modified files.

**What is the chosen solution to this problem?**

The CI prettify files, then push a new `test(ci): prettier` commit.

**Please check if the PR fulfills these requirements**
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: .travis/after_success_git.sh [hunk 1]
```diff
@@ -24,6 +24,10 @@ if [ -n "$GH_TOKEN" ]; then
 		git -c user.name="travis" -c user.email="travis" commit -m "test(ci): update code style outputs"
 		echo "✓ Commit updated lint output to $TRAVIS_PULL_REQUEST_BRANCH"
 
+		git add --all
+		git -c user.name="travis" -c user.email="travis" commit -m "test(ci): prettier"
+		echo "✓ Commit prettified files to $TRAVIS_PULL_REQUEST_BRANCH"
+
```
Comment (jmfrancois) [inline .travis/after_success_git.sh]: everything else is prettier ... not sure this is very futur proof may be just src ?

File: output/components.sasslint.txt [hunk 2]
```diff
@@ -6,11 +6,18 @@
 src/Actions/ActionFile/ActionFile.scss
   5:12  error  Strings must use single quotes  quotes
 
+src/Status/Status.scss
+  10:1  warning  Space expected between blocks  empty-line-between-blocks
+  13:1  warning  Space expected between blocks  empty-line-between-blocks
+  18:1  warning  Space expected between blocks  empty-line-between-blocks
+  24:1  warning  Space expected between blocks  empty-line-between-blocks
+  28:1  warning  Space expected between blocks  empty-line-between-blocks
```
Comment (jmfrancois) [inline output/components.sasslint.txt]: WAT ?

File: .travis/after_success_git.sh (hunk 1 above)
Comment (ncomont) [reply .travis/after_success_git.sh]: :ok_hand:

File: output/components.sasslint.txt (hunk 2 above)
Comment (ncomont) [reply output/components.sasslint.txt]: Oh, I will update `.sass-lint.yml` on the tools repository, thanks

Comment (ncomont) [reply output/components.sasslint.txt]: https://github.com/Talend/tools/pull/54 :)

File: .sass-lint.yml [hunk 3]
```diff
@@ -34,11 +34,6 @@ rules:
     - 2
     -
       include: false
-  empty-line-between-blocks:
```
Comment (jsomsanith) [inline .sass-lint.yml]: Did you remove this rule because prettier doesn't fix it ?

**Pull request merged**